### ISSUE TITLE
Questions on MS1-subtract and mass-acc-quant Options

### ISSUE THREAD

**Issue by dong-gi-jang** [vdemichev/DiaNN#952]
Hello,

I've been making great use of your program and it's been working wonderfully for me. 
I have a question though. 
I'm curious about what the MS1-subtract option and the mass-acc-quant option stand for.

Thank you,
Dong-gi Jang

**Comment by vdemichev** [comment]
Hi Dong-gi,

ms1-subtract signficantly reduces the degree of MS1 interference between isotopic channels when using --channels.
mass-acc-quant specifies which mass tolerance threshold should be used for chromatogram extraction for quantification

Best,
Vadim

**Comment by dong-gi-jang** [comment]
Thank you for the quick response!
Can I understand that MS1 interference in the ms1-subtract option is calculated using something like averagine?

Thank you again,
Dong-gi Jang

**Comment by vdemichev** [comment]
No, it's more simple, just subtracts whatever signal there is at the peptide mass minus C13/C12 delta mass.
